js/setup.js

```diff
@@ -39,12 +39,92 @@ var WIZARD_EYES_COLORS = [
   'green'
 ];
 
+var WIZARD_FIREBALL_COLORS = [
+  '#ee4830',
+  '#30a8ee',
+  '#5ce6c0',
+  '#e848d5',
+  '#e6e848'
+];
+
 var TOTAL_WIZARDS = 4;
 var wizards = [];
 
 var similarList = document.querySelector('.setup-similar-list');
 var template = document.querySelector('#similar-wizard-template').content.querySelector('.setup-similar-item');
 
+var ESC_KEY = 'Escape';
+var ENTER_KEY = 'Enter';
+
+var MIN_NAME_LENGTH = 2;
+var MAX_NAME_LENGTH = 25;
+
+var setup = document.querySelector('.setup');
+var setupOpen = document.querySelector('.setup-open');
+var setupClose = setup.querySelector('.setup-close');
+
+var userNameField = setup.querySelector('.setup-user-name');
+var coatColorField = setup.querySelector('input[name=coat-color]');
+var eyesColorField = setup.querySelector('input[name=eyes-color]');
+var fireballColorField = setup.querySelector('input[name=fireball-color]');
+
+var wizardCoat = setup.querySelector('.wizard-coat');
+var wizardEyes = setup.querySelector('.wizard-eyes');
+var wizardFireball = setup.querySelector('.setup-fireball-wrap');
+
+// закрытие окна
+var onSetupCloseClick = function () {
+  setup.classList.add('hidden');
+  document.removeEventListener('keydown', onEscKeydownClick);
```

Comment:
```suggestion
  document.removeEventListener('keydown', onDocumentKeydown);
```

on + элемент (это то, что перед словами addEventListener) + событие (это то, что в кавычках)